userguide/system.rst

```diff
@@ -1702,7 +1702,8 @@ SSH Keypairs
 SSH public and private keypairs in
 :menuselection:`System --> SSH Keypairs`.
 These are generally used when configuring :ref:`SSH Connections` or
-*SFTP* :ref:`Cloud Credentials`.
+*SFTP* :ref:`Cloud Credentials`. Encrypted keypairs or keypairs with
+passphrases are not supported.
```

Comment:
Can you have a non-encrypted keypair with a passphrase? If so, then LGTM. Otherwise, I would say it could be reduced to
```
Encrypted keypairs are not supported.
```

Reply:
hm, good question - I'll look into it a bit more before merging.

Reply:
Looks like passphrase is only used in conjunction with encryption: https://www.ssh.com/ssh/passphrase - will rework text.

Reply:
Passphrases imply encryption.  But the reader doesn't necessarily know that, which was why this was worded to include both.  Please go back to that wording.